mifospay/src/main/res/layout/activity_mobile_verification.xml

```diff
@@ -91,13 +91,13 @@
 
             <android.support.design.widget.TextInputLayout
                 style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
-                android:layout_width="@dimen/value_160dp"
+                android:layout_width="@dimen/value_150dp"
```

Comment:
Changing the dimensions is not a reliable solution as this can again fail for some device with a thinner aspect ratio. The proper way to handle this in my opinion is to use _layout weights_

Reply:
Layout weights are applied.

<img src="https://user-images.githubusercontent.com/73793609/162576968-d692f20e-bcf8-4681-8f52-3604215cab7a.png" weight="250" height="500"/>